src/compiler/crystal/macros/methods.cr

```diff
@@ -723,6 +729,8 @@ module Crystal
       case method
       when "==", "!=", "stringify", "class_name"
         return super
+      when "symbolize"
+        return self
```

Comment:
@kirbyfan64 This is the only condition I'm not sure about. Why not call `super` here?